bin/docker-start

```diff
@@ -10,4 +10,8 @@ done
 bundle exec rake db:migrate
 rm -rf tmp/pids
 
-exec rails s -b 0.0.0.0
+if [ "${OCTOBOX_SIDEKIQ_SCHEDULE_ENABLED}" == "true" ]; then
```

Comment:
Good point. I use now the definition of 'boolness' which I found here https://github.com/octobox/octobox/blob/b1459701d753a5ececc922c3a05bf6824585d961/lib/octobox/configurator.rb#L161-L163 i.e. true for "true" and "1", false otherwise.